app/models/podcast_import.rb

```diff
@@ -301,6 +283,35 @@ def create_story(entry, series)
     story
   end
 
+  def get_or_create_template(segments)
+    num_segments = [segments.to_i, 1].max
+    if !templates[num_segments]
+      template = series.audio_version_templates.create!(
+        label: "Podcast Audio #{num_segments}",
```

Comment:
Might confuse users to see 'Podcast Audio 3' as their template label if it's their only one -- if it were me, I'd wonder where versions 1 and 2 went. How about `"Podcast Audio #{num_segments} Segments"` ?

Reply:
good suggestion, will do.